tasks/commandLineArguments.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as minimist from 'minimist';
+import * as path from 'path';
+
+let argv = minimist(process.argv.slice(2), { 
+    boolean: ['retainVsix']
+});
+
+export const commandLineOptions ={
+    retainVsix: !!argv['retainVsix'],
+    vsixPackagingOutputFolder:  makePathAbsolute(argv['o']),
```

Comment:
Should we name this just as an output folder so that any other gulp tasks(that may or may not be related to vsixs) can also specify a `-o` option to list their output folder.